script/UI/Container.lua

```diff
@@ -23,328 +23,337 @@ Container.name        = 'Container'
 
 -- Container Functions ---------------------------------------------------------
 
-function Container:onFindMouseFocusChildren (mx, my, foci)   end
-function Container:onInputChildren          (state)          end
-function Container:onUpdateChildren         (state)          end
-function Container:onLayoutSizeChildren     ()               end
-function Container:onLayoutPosChildren      ()               end
-function Container:onDrawChildren           (focus, active)  end
-function Container:onDrawDebugChildren      (focus, active)  end
-
-function Container:setEState                (eState)         end
-function Container:setEnabled               (enabled, speed) end
-function Container:toggleEnabled            (speed)          end
-function Container:isEnabled                ()               end
-function Container:enable                   (speed)          end
-function Container:disable                  (speed)          end
-function Container:completeFade             ()               end
-
-function Container:onEnable                 ()               end
-function Container:onEnableDone             ()               end
-function Container:onDisable                ()               end
-function Container:onDisableDone            ()               end
-
-function Container:add                      (child, enabled) end
-function Container:remove                   (child)          end
-function Container:addImmediate             (child)          end
-function Container:removeImmediate          (child)          end
+function Container:onFindMouseFocusChildren(mx, my, foci) end
```

Comment:
Is it possible to avoid somewhere in the configuration adding new lines if function is on one line?
We lost here nice grouping

Reply:
done :D

Reply:
added